api/api/validator.py

```diff
@@ -129,6 +130,7 @@
                 "max_login_attempts": {"type": "integer"},
                 "block_time": {"type": "integer"},
                 "max_request_per_minute": {"type": "integer"},
+                "max_memory_usage": {"type": "integer", "minimum": MIN_VALUE_MAX_MEMORY_USAGE},
```

Comment:
I have been researching if it is possible to create a condition in JSONschema where we allow to use the value 0 (no limit on memory usage) or a minimum (`max_memory_usage = 0 or max_memory_usage >= MIN_VALUE_MAX_MEMORY_USAGE`), but it doesn't seem feasible. The limitation on the `validator.py` side will have to be removed again:

```suggestion
                "max_memory_usage": {"type": "integer"},
```